src/tests.rs

```diff
@@ -2236,6 +2236,95 @@ export const tpl2 = `Value: ${num}`;
     ]
   );
 
+  json_test!(export_const_destructured,
+    r#"const c = { a: "a", b: 2 };
+    const d: { f: string; g: number; } = { f: "f", g: 2 };
+
+    export const { a, b }: {
+      /** type alias doc */
```

Comment:
The tsc language server associates JSDoc in the type with the variable, so I would expect `a` and `b` to have JSDoc associated with them.  I haven't added it to the fixture yet.